src/components/myFooter/MyFooter.styles.jsx

```diff
@@ -0,0 +1,101 @@
+import styled from 'styled-components'
+import tw from 'twin.macro'
+
+export const Footer = styled.div`
+  ${tw`
+    h-[774px] custom:h-[483px] bg-brown-4 font-Poppins flex flex-col justify-between  
+  `}
+`
+
+export const FooterTop = styled.div`
+  ${tw`
+    px-[24px] pt-[23px] pb-[49px] custom:h-[400.3px] custom:px-[39px] custom:py-[64px] flex justify-between flex-col gap-[47px] custom:flex-row custom:gap-[0px]
```

Comment:
custom (1450px) is too wide to add flex-col
![image](https://github.com/dscnitrourkela/project-vanilla/assets/68679980/332ade24-02ed-4174-975c-6a06fcc9681b)
make it flex-row till min 680px